### PR TITLE
SRS GHRSST - add path creation for VIIRS and multisensor data

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

File: aodndata/srs/srs_ghrsst.py [hunk 1]
```diff
@@ -10,20 +10,45 @@
...
                                 (?P<nc_time_cov_start>[0-9]{14})-ABOM-
                                 (?P<product_type>L3U)_.*-AVHRR
-                                (?P<sat_number>[0-9].*)_D-
+                                (?P<sat_value>[0-9].*)_D-
                                 (?P<pass_direction>Asc|Des)
                                 (?P<optional>_Southern|)
                                 \.nc$
                                 """, re.VERBOSE)
 
+L3U_VIRRS_FILE_PATTERN = re.compile(r"""
```
Comment (bpasquer) [inline aodndata/srs/srs_ghrsst.py]: There is a difference in the file pattern : is it VIRRS or VIIRS?

File: aodndata/srs/srs_ghrsst.py [hunk 2]
```diff
@@ -10,20 +10,45 @@
...
+                                (?P<nc_time_cov_start>[0-9]{14})-ABOM-
+                                (?P<product_type>L3U)_.*-VIIRS_NPP.*
+                                (?P<optional>_Southern|)
+                                \.nc$
+                                """, re.VERBOSE)
+
+L3C_VIRRS_FILE_PATTERN = re.compile(r"""
```
Comment (bpasquer) [inline aodndata/srs/srs_ghrsst.py]: same as above

File: aodndata/srs/srs_ghrsst.py (hunk 1 above)
Comment (lbesnard) [reply aodndata/srs/srs_ghrsst.py]: thanks. will correct. it's viirs

Comment (lbesnard) [reply aodndata/srs/srs_ghrsst.py]: @bpasquer done !

**Pull request merged**